backend/src/repos/BuildingRepo.js

```diff
@@ -0,0 +1,25 @@
+import QueryHandler from './QueryHandler';
+
+export class BuildingRepo extends QueryHandler {
+
+    constructor(db,errorCodes) {
+        super(db,errorCodes);
+    };
+
+    async add({kingdomId,type,level,hp,started_at,finished_at}) {
+        if (!kingdomId) throw new Error(this.errorCodes.missingKingdomId);
+        const query = this.validateQuery`
+            INSERT INTO buildings 
+            (kingdom_id,type,level,hp,started_at,finished_at) 
+            VALUES(${kingdomId},${type},${level},${hp},${started_at},${finished_at})
+        `;
+        return await (this.sendQuery(query));
```

Comment:
@gulyaasjaanos you don't have to await the sendQuery here nor in any other place, since you're going to await the result in the services anyway